test/test.code-workspace

```diff
@@ -1,5 +1,8 @@
 {
     "folders": [
+        {
+            "path": "./nightly/testFolder/dotnet-hello-world"
```

Comment:
I would suggest putting this project in a "2.1" folder since we'll likely want to test a "3.1" project in the future

"./nightly/testFolder/dotnet-hello-world/2.1"